### ISSUE TITLE
How to configure

### ISSUE THREAD

**Issue by guohuihot** [vuejs/eslint-plugin-vue#954]
**Please describe what the rule should do:**

**What category should the rule belong to?**

- [ ] Enforces code style
- [ ] Warns about a potential error
- [ ] Suggests an alternate way of doing something
- [ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**
```vue

```

**Additional context**
How to configure？
vue/component-name-in-template-casing
![image](https://user-images.githubusercontent.com/3119995/64666292-56e50080-d488-11e9-98a1-c0244912b405.png)

**Comment by ota-meshi** [comment]
Hello.

Do you want to know how to configure of `vue/component-name-in-template-casing`?
If so, please refer to the documentation.

https://eslint.vuejs.org/rules/component-name-in-template-casing.html
